scripts/md2rst.py

```diff
@@ -27,7 +27,10 @@ def download_pandoc():
 
 
 def md2rst(text):
+    input_format = os.environ.get('FORMAT', 'markdown_github')
     download_pandoc()
-    output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('scripts', 'filters', 'yml_filter.py')])
```

Comment:
sorry, I didn't realize this was a line continuation. I would suggest to store the result of `os.path.join` in a seperate variable and then pass it to pypandoc.
```python
yml_filter = os.path.join('scripts', 'filters', 'yml_filter.py')
pypandoc.convert_text(text, 'rst', format=input_format,
                      filters=[yml_filter])
```
You can read the Indentation [section](https://www.python.org/dev/peps/pep-0008/#id17) of PEP8. Most python linters follow the guidelines specified there.